tests/phpstan/bootstrap.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
Lack a tests/index.php & tests/phpstan/index.php

Reply:
This directory is removed on Build release